app/src/main/java/org/coepi/android/cen/CenManager.kt

```diff
@@ -25,11 +25,11 @@ class CenManager(
         disposables += Observables.combineLatest(
             blePreconditions.bleEnabled,
             // Take the first CEN, needed to start the service
-            cenRepo.CEN
+            cenRepo.cen
         )
         .take(1)
         .subscribeBy(onNext = { (_, firstCen) ->
-            bleManager.startService(firstCen.toString()) // TODO review String <-> ByteArray
+            bleManager.startService(firstCen) // TODO review String <-> ByteArray
```

Comment:
Is the comment still applicable since changing to the cen object?

Reply:
This is solved now 👍